semantic/src/site/globals/site.overrides

```diff
@@ -2780,16 +2779,47 @@ h6:first-child {
       padding: 0.5em 0.5em;
     }
     .stage-label {
-      font-size: 0.57rem;
+      font-size: 0.7rem;
+    }    
+  }
+  #overview .left-item {
+    min-width: unset;
+  }
+  #timeline {
+    margin-left: 8px;
+    margin-right: 8px;
+  }
+  
+  #documents-tab {
+    //smaller icon in Review documents tab
+    .ui.image {
+      max-height: 60px;
+    }
+    .flex-extra {
+      gap: 5px;
+    }
+    .ui.list {
+      margin-top: 0px;
+    }
+    .ui.list > .item {
+      margin-top: 10px;
+      padding: 10px;
     }
   }
 
   #notes-tab .options-row {
     flex-direction: column;
+    align-items: flex-start;
```

Comment:
Can you just add some left padding to make this row flush with the stuff beneath it?
![Screenshot 2023-11-07 at 4 59 25 PM](https://github.com/msupply-foundation/conforma-web-app/assets/5456533/f98dec02-2b01-4093-91da-48c01f046984)